src/book/02-system/02-subsystems/04-motion.mdx

```diff
@@ -6,17 +6,70 @@ description: Motion and how it works in the NUbots codebase.
+| -------------------- | ---------------------------------------------------------------------------------------- |
+| IS_DOUBLE_SUPPORT    | Splines for when the robot is in double support phase, i.e. both feet are on the ground. |
+| IS_LEFT_SUPPORT_FOOT | Splines for the support foot.                                                            |
+| TRUNK_POS_X          | The Cartesian position of the robot's torso in the x-axis.                               |
+| TRUNK_POS_Y          | The Cartesian position of the robot's torso in the y-axis.                               |
+| TRUNK_POS_Z          | The Cartesian position of the robot's torso in the z-axis.                               |
+| TRUNK_AXIS_X         | The orientation - x-axis of the robot's torso.                                           |
+| TRUNK_AXIS_Y         | The orientation - y-axis of the robot's torso.                                           |
+| TRUNK_AXIS_Z         | The orientation - z-axis of the robot's torso.                                           |
+| FOOT_POS_X           | The Cartesian position of the robot's flying foot in the x-axis.                         |
+| FOOT_POS_Y           | The Cartesian position of the robot's flying foot in the y-axis.                         |
+| FOOT_POS_Z           | The Cartesian position of the robot's flying foot in the z-axis.                         |
+| FOOT_AXIS_X          | The orientation - x-axis of the robot's flying foot.                                     |
+| FOOT_AXIS_Y          | The orientation - y-axis of the robot's flying foot.                                     |
+| FOOT_AXIS_Z          | The orientation - z-axis of the robot's flying foot.                                     |
+
+The points that make-up all these splines are in the [WalkEngine.cpp file](https://github.com/NUbots/NUbots/blob/ff2cf1f7f38bf558494e305a99544e15affaf719/module/motion/QuinticWalk/src/engine/WalkEngine.cpp#L331). The [configuration parameters](https://github.com/NUbots/NUbots/blob/biddulph/quintic_walk/module/motion/QuinticWalk/data/config/QuinticWalk.yaml) affect some of these points. Values are all in [SI units](https://en.wikipedia.org/wiki/International_System_of_Units).
+
+The motion subsystem contains modules responsible for walking, kicking, moving the head, and running scripts. They perform these motions by sending servo commands (called targets) to the motors.
+
+## Script Engine
 
 Scripts are static motions for the robot. They specify what joint angles to move to and how long the robot should take to get to those joint angles. For example, standing up is a script telling the robot to move its joints to the stand position over one second. There can be many of these position specifications in sequence to make the robot do more complex movements like getting up or kicking.
 
 To learn how to tune scripts, see the [ScriptTuner guide](/guides/main/tuning-and-running-scripts#script-tuner).
 
-## Messages
+### Messages
```

Comment:
Github wont let me put a comment on the exact line for this, but `ExecuteScriptByName` takes a `duration_modifier` not a `duration`. Same for `ExecuteScript`.

Reply:
It won't let you because it's not a change in this PR. Script engine was a different PR, this PR is just about the walk. Any change should be in a new PR.